webrtc.html

```diff
@@ -8857,6 +8859,58 @@ <h2 id="sec.stats-model">Statistics Model</h2>
       [[!WEBRTC-STATS]].</p>
     </section>
     <section>
+      <h3>The stats selection algorithm</h3>
+      <p>
+	The <dfn>stats selection algorithm</dfn> is as follows:
+	<ul>
+          <li>
+            Let <var>result</var> be an empty RTCStatsReport.
+          </li>
+	  <li>If the selector is a MediaStreamTrack, add the following
+            objects to <var>result</var>, if they exist:
+	    <ul>
+	      <li>
+		The RTCMediaStreamTrackStats corresponding to the selector.
+	      </li>
+	      <li>
+		If the selector track is inbound, the RTCInboundRTPStreamStats
+                corresponding to the selector that has "remote=false", and the
+                RTCOutboundRTPStreamStats corresponding to the selector that has
+                "remote=true".
```

Comment:
I'd phrase that as "the information we have about the other end's view of the connection is stored in the item you find through associateStatsId. Its values are derived from data received via RTCP."

@jan-ivar you left out the step of going from the MediaStreamTrackStats to the RTCRtpStreamStats.